src/main/java/ui/windows/CardDisplayData.java

```diff
@@ -0,0 +1,28 @@
+package ui.windows;
+
+import enums.Rank;
+import enums.Suit;
+
+public class CardDisplayData {
+    private Suit suit;
+    private Rank rank;
+    private boolean justDrawn;
+
+    public CardDisplayData(Suit suit, Rank rank) {
+        this.suit = suit;
+        this.rank = rank;
+        justDrawn = false;
```

Comment:
I'd add some documentation here just to know how this class meshes with the other classes.